{{cookiecutter.project_slug}}/notebooks/Untitled.ipynb

```diff
@@ -0,0 +1,49 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# change working directory to the project root:\n",
+    "%cd /project\n",
```

Comment:
Note: here we explicitly change notebook's working directory to the project root, because the notebook still works in directory `{project_root}/notebooks` even if we started it with `--notebook-dir=$(PROJECT_PATH_ENV)`:

Firstly, the jupyter does start in the project root (`/project`), but then, the IPython kernel changes its directory to `/project/notebooks`:
```
[I 11:57:22.720 NotebookApp] Serving notebooks from local directory: /project
...
[D 11:57:37.078 NotebookApp] Kernel args: {'kernel_name': 'python3', 'cwd': '/project/notebooks'}
```
I agree that `%cd /project` in the top of jupyter notebooks looks bad, but I could not find the place where the kernel `python3` takes its `cwd` in the log above.

Reply:
Unfortunately, I could not find the solution for this problem (asked in jupyter project: https://github.com/jupyter/notebook/issues/4924). Here (https://stackoverflow.com/questions/15680463/change-ipython-jupyter-notebook-working-directory) we are also advised to use `%cd`.